go.mod

```diff
@@ -146,6 +175,7 @@ require (
 	golang.org/x/sys v0.11.0 // indirect
 	golang.org/x/text v0.11.0 // indirect
 	golang.org/x/time v0.3.0 // indirect
+	gonum.org/v1/gonum v0.13.0 // indirect
```

Comment:
I looked at the failure. I think this can be ignored for now; it says _A tiled image with a height of 0 and a very large width can cause excessive CPU consumption, despite the image size (width * height) appearing to be zero_

And the suggestion is to _Upgrade golang.org/x/image/tiff to version 0.10.0 or higher_